changes in JOSS paper (#218)
precice/aste: docs/paper.md

# Summary

Simulating multi-physics phenomena for real-world applications states various challenges in scientific computing.

> The behavior of individual physical domains involved are often described through distinct partial-differential equations, which need to be solved in a certain subdomain.

Reviewer: The verb is supposed to refer to "The behavior", which is singular. So I would expect `is` to be the correct grammar here, no?

Author: To me, the subject is the behavior of the (multiple) domains, which I see as plural. If this was the behavior of a single domain, I would agree with you.

Reviewer: Overall, there are multiple physical domains involved. In the sentence above, we refer to each individual domain, i.e., one behavior per domain. Honestly speaking, I am not a native English speaker, so I am not entirely sure what's correct in this scenario. If you understand it in this way as plural, I am good to go with 'are'.

Author: What about
> Each individual physical domain has behavior that is often described through a distinct set of partial-differential equations that needs to be solved in that domain.

Reviewer: Perfect! I propagated the rephrasing also to the subsequent sentence.

> The behavior of individual physical domains involved are often described through distinct partial-differential equations, which need to be solved in a certain subdomain.
> Their interaction is then achieved through bidirectional exchange of suitable coupling data between all subdomains.

Reviewer: ```suggestion
Each individual physical domain has behavior that is often described through a distinct set of partial-differential equations that needs to be solved in that domain.
Their interaction is then achieved through bidirectional exchange of suitable coupling data between all involved domains.
```
